pxemgr/filemanager.go

```diff
@@ -0,0 +1,41 @@
+package pxemgr
+
+import (
+	"bytes"
+	"encoding/base64"
+	"github.com/golang/glog"
+	"io/ioutil"
+	"path"
+	"text/template"
+)
+
+type Files map[string]string
+
+func (mgr *pxeManagerT) RenderFiles(ctx interface{}) *Files {
```

Comment:
Please execute `gofmt`. Ideally we also need `golint`, but i'm ok to do it separately

Reply:
`gofmt` is needed as CI won't succeed unless code is `gofmt`ed
what looks funky? the empty lines?